applications/vila_live/tests/test_tinychat.py

```
@@ -75,54 +70,130 @@ def setUpClass(cls):
+                    return models[0]
+            except (requests.RequestException, ValueError) as error:
+                last_error = error
+            time.sleep(5)
+
+        raise RuntimeError(f"Worker did not register within 300 seconds: {last_error}")
+
+    @staticmethod
+    def _load_test_frame():
+        """Extract one JPEG frame from the test video and return it as base64."""
+        video_path = os.path.join(os.environ["HOLOHUB_DATA_DIR"], "vila_live", "meeting.mp4")
+        frame = subprocess.check_output(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)